### PR TITLE
Found and removed dead code. 

### PR DESCRIPTION
**Pull request by zenspider**
Testing out changes to debride to work on erb code. Found some dead code.

Please review @phiggins and @thagomizer. I used `rake routes` to generate my whitelist, which potentially has false positives too. Whitelist below:

```
bulk_action
bulk_delete
checklist
create
dashboard
delete
destroy
edit
export
history_index
history_show
index
join_us
new
past
projects
show
show_in_app
subscribe
update
# manually added:
validate_each
get_event
update_published_at
set_title
verify_password
```

### CODE REVIEW

File: app/models/talk.rb [hunk 1]
```diff
@@ -33,12 +33,4 @@ def self.older
                            :message => "%{value} is not a valid talk kind"
                           })
   validates :title, :presenter, :email, :presence => true
-
-  def scheduled?
-    self.scheduled_date
-  end
-
-  def kind_enum
```
Comment (thagomizer) [inline app/models/talk.rb]: This might be used by rails_admin. TODO verify that.

**Comment by zenspider** [comment]
New push. Please review @thagomizer and @phiggins

**Comment by phiggins** [comment]
Seems good to me. :+1:

File: app/views/talks/checklist.html.erb [hunk 2]
```diff
@@ -1,64 +0,0 @@
...
-    within walking distance, but a pain to carry a keg from if
-    walking). It runs about $90, plus a $150 refundable deposit when
-    you bring the keg back. I think either that or canned/bottled beer
-    would be fine - we didn't finish it.
-  </li>
-
-  <li>
-    Ping <a href="pete@peterhiggins.org">Pete Higgins</a>, who is the
-    contact at Substantial, about setup and/or anything else related
-    to the venue.
```
Comment (phiggins) [inline app/views/talks/checklist.html.erb]: Haha, just a bit out of date...

**Pull request merged**